### PR TITLE
Update polkadot config

### PR DESCRIPTION
**Pull request by emresurmeli**
Updating the polkadot.json to match docusaurus.json format. Please review to make sure things are looking right.
We recently upgraded to Docusaurus v2 and our search is broken due to indexing issues we believe, this should fix things. If there is anything else that needs to be looked at in order to migrate successfully to v2 please let us know.

### CODE REVIEW

File: configs/polkadot.json [hunk 1]
```diff
@@ -11,29 +11,39 @@
...
+      "hierarchy",
+      "content",
+      "anchor",
+      "url",
+      "url_without_anchor",
+      "type"
     ]
   },
   "conversation_id": [
-    "969238060"
```
Comment (shortcuts) [inline configs/polkadot.json:36]: The conversation ID should stay the same

File: configs/polkadot.json [hunk 2]
```diff
@@ -11,29 +11,39 @@
...
+    "833762294"
```
Comment (shortcuts) [inline configs/polkadot.json:46]:
```suggestion
    "969238060"
```

The conversation ID should stay the same

Comment (emresurmeli) [reply configs/polkadot.json]: Thank you, fixed

**Comment by nam-hle** [comment]
Hi @emresurmeli, please help me adjust the `start_urls` to the direct one: `https://wiki.polkadot.network/docs/getting-started`. Thank a lot.

**Comment by shortcuts** [comment]
Thanks for updating @emresurmeli, please merge the master branch so that I can merge the PR

**Pull request merged**